### ISSUE TITLE
Handle /* */ style comments

### ISSUE THREAD

**Issue by skuroda** [CodeEffect/BoundKeys#1]
These are "valid" in ST, so don't forget to remove them also. 

Not really an issue, but you might want to consider creating a quick panel entry also. From writing my key conflict detection plugin, it's not always necessary to bind it to a key as it's not a common operation. I did see the menu entry, but I think it would be nice to have a quick panel entry also.  Anyways, just my 2 cents.

**Comment by CodeEffect** [comment]
Well spotted, thank you! Hadn't thought of those (well they didn't come up in any of my files which was the reason I had to strip the "//" comments). 

Nice idea on the quick panel too, I assumed we got one of those for free, didn't realise we had to add. Agreed that it possibly doesn't warrant a menu item / key binding. I may remove those and add them as an example file. 

Cheers for having a gander over it :)
